atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -552,6 +551,48 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
         return transactionManager;
     }
 
+    private TimeLockFeedbackBackgroundTask getTimeLockFeedbackBackgroundTask(
+            @Output List<AutoCloseable> closeables,
+            AtlasDbConfig config,
+            Refreshable<AtlasDbRuntimeConfig> runtimeConfig) {
+        Supplier<List<TimeLockClientFeedbackService>> timeLockClientFeedbackServicesSupplier
+                = getTimeLockClientFeedbackServices(config,
+                runtimeConfig, userAgent());
+        return initializeCloseable(
+                () -> TimeLockFeedbackBackgroundTask.create(
+                        globalTaggedMetricRegistry(),
+                        () -> AtlasDbVersion.readVersion(),
+                        serviceName(),
+                        timeLockClientFeedbackServicesSupplier), closeables);
+    }
+
+    @VisibleForTesting
+    static Supplier<List<TimeLockClientFeedbackService>> getTimeLockClientFeedbackServices(AtlasDbConfig config,
+            Refreshable<AtlasDbRuntimeConfig> runtimeConfig,
+            UserAgent userAgent) {
+        Refreshable<ServerListConfig> serverListConfigSupplier =
+                getServerListConfigSupplierForTimeLock(config, runtimeConfig);
+        return new CachedTransformingSupplier<>(
+                serverListConfigSupplier,
+                serverListConfig -> createProxiesForFeedbackService(userAgent, serverListConfig));
+    }
+
+    private static List<TimeLockClientFeedbackService> createProxiesForFeedbackService(UserAgent userAgent,
+            ServerListConfig serverListConfig) {
+        return serverListConfig.servers().stream()
+           .map(uri -> AtlasDbHttpClients.createProxy(
+                   serverListConfig.trustContext(),
+                   uri,
+                   TimeLockClientFeedbackService.class,
+                   AuxiliaryRemotingParameters.builder()
+                           .shouldUseExtendedTimeout(true)
+                           .shouldLimitPayload(true)
+                           .shouldRetry(true)
+                           .userAgent(userAgent)
+                           .build()))
```

Comment:
This is fine, but we should look at using the new Dialogue services / AtlasDbDialogueServiceProvider.